api/usersd/pb/usersd.proto

```diff
@@ -191,4 +235,10 @@ service APIService {
     // Archive Retrieval
     rpc ArchiveRetrievalLs(ArchiveRetrievalLsRequest) returns (ArchiveRetrievalLsResponse){}
     rpc ArchiveRetrievalLogs(ArchiveRetrievalLogsRequest) returns (stream ArchiveRetrievalLogsResponse) {}
+
+    // Fil Rewards
+    rpc ListFilRewards(ListFilRewardsRequest) returns (ListFilRewardsResponse) {}
+    rpc ClaimFil(ClaimFilRequest) returns (ClaimFilResponse) {}
+    rpc ListFilClaims(ListFilClaimsRequest) returns (ListFilClaimsResponse) {}
+    rpc FilRewardsBalance(FilRewardsBalanceRequest) returns (FilRewardsBalanceResponse) {}
```

Comment:
@andrewxhill check out the user api versions of the fil rewards API. It is very similar to the filrewards service, but infers certain information from the auth context (org and dev) instead of allowing the values to be provided by the client.

Reply:
looks solid